single_stage_detector/README.md

```diff
@@ -0,0 +1,293 @@
+
+- [Summary](#summary)
+- [Running the reference](#running-the-reference)
+  - [Prerequisites](#prerequisites)
+  - [Building the docker image](#building-the-docker-image)
+  - [Download dataset](#download-dataset)
+  - [Download the pretrained backbone](#download-the-pretrained-backbone)
+  - [Training on NVIDIA DGX-A100 (single node)](#training-on-nvidia-dgx-a100-single-node)
+  - [Training on NVIDIA DGX-A100 (multi node)](#training-on-nvidia-dgx-a100-multi-node)
+  - [Hyperparameter settings](#hyperparameter-settings)
+- [Dataset/Environment](#datasetenvironment)
+  - [Publication/Attribution](#publicationattribution)
+  - [The MLPerf Subset](#the-mlperf-subset)
+- [Model](#model)
+  - [Backbone](#backbone)
+  - [Head network](#head-network)
+  - [Detection heads and anchors](#detection-heads-and-anchors)
+  - [Ground truth and loss function](#ground-truth-and-loss-function)
+  - [Input augmentations](#input-augmentations)
+  - [Publication/Attribution](#publicationattribution-1)
+- [Quality](#quality)
+  - [Quality metric](#quality-metric)
+  - [Quality target](#quality-target)
+  - [Evaluation frequency](#evaluation-frequency)
+  - [Evaluation thoroughness](#evaluation-thoroughness)
+
+# Summary
+Single Shot MultiBox Detector (SSD) is an object detection network. For an
+input image, the network outputs a set of bounding boxes around the detected
+objects, along with their classes. For example:
+
+![](https://upload.wikimedia.org/wikipedia/commons/3/38/Detected-with-YOLO--Schreibtisch-mit-Objekten.jpg)
+
+SSD is a one-stage detector, both localization and classification are done in a
+single pass of the network. This allows for a faster inference than region
+proposal network (RPN) based networks, making it more suited for real time
+applications like automotive and low power devices like mobile phones. This is
+also sometimes referred to as being a "single shot" detector for inference.
+
+# Running the reference
+The reference code is intended to run on NVIDIA GPUs, it is tested on A100 and
+V100 cards but other GPUs should work too.
+
+## Prerequisites
+The recommended way to run the reference is with docker. You need to setup your
+machine with:
+1. NVIDIA drivers. Can be installed with [CUDA toolkit](https://developer.nvidia.com/cuda-downloads).
+2. [Docker](https://docs.docker.com/engine/install/)
+3. [NVIDIA container runtime](https://github.com/NVIDIA/nvidia-docker)
+
+If you are using Ubuntu 20.04, you can install these components by running:
+```bash
+bash ./install_cuda_docker.sh
+```
+
+In addition to the above requirements, it's recommended to use
+[Slurm](https://developer.nvidia.com/slurm) for Multi-node training
+(the installation and configuration of Slurm is cluster depended and not
+covered in this README).
+
+## Building the docker image
+Once the prerequisites are met, you can build the benchmark docker image with:
+```bash
+cd ./single_stage_detector/
+docker build -t mlperf/single_stage_detector .
```

Comment:
shouldn't a docker run or launch command be provided to run the container after build?

Reply:
I added instructions on how to use the image with the single-node config.